aconfig/aconfig.py

```diff
@@ -11,16 +11,16 @@
 import os
 import re
 import copy
-import typing
+from typing import Optional, Any, Type
 
 from yaml.representer import SafeRepresenter
 import yaml
 
 
-class AttributeAccessDict(dict):
+class AttributeAccessDict(dict[str, Any]):
     '''Wrapper around Python dict to make it accessible like an object.
     '''
-    def __init__(self, input_map):
+    def __init__(self, input_map:dict[str, Any]):
```

Comment:
NIT here and in many other places. Let's add a ` ` (space) between the `:` and the hint

Reply:
Also, let's use `Dict` instead of `dict` for compatibility